src/Nri/Ui/Tooltip/V3.elm

```diff
@@ -835,6 +836,12 @@ auxiliaryDescription =
     Attribute (\config -> { config | purpose = AuxillaryDescription })
 
 
+{-| -}
+helpfullyDisabled : Attribute msg
+helpfullyDisabled =
+    Attribute (\config -> { config | purpose = HelpfullyDisabled })
```

Comment:
Maybe the `purpose` should be `AuxillaryDescription`? Or do we intend for the behavior to differ at any point between AuxillaryDescription and HelpfullyDisabled?